Modules/CustomRolesHelper.cs

```diff
@@ -1392,8 +1407,8 @@ public static CountTypes GetCountTypes(this CustomRoles role)
            CustomRoles.Shroud => CountTypes.Shroud,
            CustomRoles.Werewolf => CountTypes.Werewolf,
            CustomRoles.Wraith => CountTypes.Wraith,
-           CustomRoles.Pestilence => CountTypes.Pestilence,
-           CustomRoles.PlagueBearer => CountTypes.PlagueBearer,
+           CustomRoles.Pestilence or CustomRoles.PlagueBearer or CustomRoles.SoulCollector or CustomRoles.Death or CustomRoles.Baker or CustomRoles.Famine or CustomRoles.Berserker or CustomRoles.War 
+           => CountTypes.Apocalypse,
```

Comment:
you should be able to do a epic gamer trick here and simply do "var role when role.IsApocalypse() => CountTypes.Apocalypse,"
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/b3d8119e-b490-4504-b3e2-35bb1e7b074f)